java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -726,6 +727,23 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_stringConcatenation(JNI
   CATCH_STD(env, 0);
 }
 
+JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_interleaveColumns(JNIEnv *env, jobject j_object,
```

Comment:
This is a static method rather than an instance method.
```suggestion
JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_interleaveColumns(JNIEnv *env, jclass,
```

Reply:
Interesting. I copied this from stringConcatenation which is also a static method, so maybe that one is incorrect as well?

Reply:
Probably.  It's an ignored parameter that has the same essential type (both are pointers), so not surprising it still works.